newspaper/images.py

```diff
@@ -25,6 +27,8 @@
 thumbnail_size = 90, 90
 minimal_area = 5000
 
+BAD_URLS = ['logo', 'icon', 'banner', 'ribbon', 'sprite', 'qrcode']
+ALLOWED_TYPES = ['jpg', 'jpeg', 'png']
```

Comment:
should .gif be allowed? normally its associated with logos or regular page assets

Reply:
webp?

Reply:
Let's leave gifs out for now - good call. Can we be careful about what is in ALLOWED_TYPES, maybe also add TIFF, EPS, and RAW? Also maybe webp like vitaly suggested above, I'm not sure what image files are popular nowadays.
http://guides.lib.umich.edu/c.php?g=282942&p=1885348

Reply:
from my reading I don't think RAW or EPS are supported in browsers. There are website that purport to allow viewing of those. 
https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support
WebP seems good. TIFF seems rarely supported but we could add it